charts/seed-bootstrap/charts/vpa/values.yaml

```diff
@@ -8,6 +8,9 @@ global:
 labels:
   garden.sapcloud.io/role: vpa
 
+deploymentLabels:
+  gardener.cloud/role: seed
```

Comment:
We don't need this. Please adapt the 

```
{{ toYaml .Values.deploymentLabels | indent 4 }}
```

usage with 

```
{{- if .Values.deploymentLabels }}
{{ toYaml .Values.deploymentLabels | indent 4 }}
{{- end }}
```

to make helm3 happy.